app/screens/channel/index.tsx

```diff
@@ -1,28 +1,32 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import React from 'react';
-import {SafeAreaView, ScrollView, StatusBar, StyleSheet, Text, View} from 'react-native';
-import {
-    Colors,
-    DebugInstructions,
-    Header,
-    LearnMoreLinks,
-    ReloadInstructions,
-} from 'react-native/Libraries/NewAppScreen';
-
-import type {LaunchProps} from '@typings/launch';
-
-import {logout} from '@actions/remote/general';
-import {useServerUrl} from '@context/server_url';
+import {Database} from '@nozbe/watermelondb';
+import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
+import withObservables from '@nozbe/with-observables';
+import React, {useEffect} from 'react';
+import {useIntl} from 'react-intl';
+import {SafeAreaView} from 'react-native-safe-area-context';
+
+import StatusBar from '@components/status_bar';
+import ViewTypes from '@constants/view';
+import {MM_TABLES} from '@constants/database';
+import {isMinimumServerVersion} from '@utils/helpers';
+import {makeStyleSheetFromTheme} from '@utils/theme';
+import {unsupportedServer} from '@utils/supported_server/supported_server';
+
+import ChannelNavBar from './channel_nav_bar';
+
+import type ChannelModel from '@typings/database/models/servers/channel';
+import type SystemModel from '@typings/database/models/servers/system';
+import type UserModel from '@typings/database/models/servers/user';
+import type {LaunchType} from '@typings/launch';
 import {useTheme} from '@context/theme';
 
-type ChannelProps = LaunchProps;
+const {SERVER: {CHANNEL, SYSTEM, USER}} = MM_TABLES;
 
-const Channel = (props: ChannelProps) => {
+const Channel = ({channelRecord: channel, userRecord: user, configRecord: config, currentUserIdRecord}: ChannelProps) => {
```

Comment:
don't really like the use of `record` leave it without

Reply:
the `record` parts have been removed.